docs/InjiWalletSequence.md

```diff
@@ -0,0 +1,50 @@
+# **Understanding the workflow**
+
+### **Inji Wallet**
+
+- The use installs the application on Android or iOS device.
+  - Application link is available in our sandbox env https://collab.mosip.net/
+- After installation, user will be asked to select unlock mehtod as passcode or biometric to add security to the applicaiton
+- After selecting unlock method, user will be navigated to Home screen.
+- There is plus icon on home screen, after tapping on that icon, it will display list of issuers supported by Inji wallet to downlaod credentials.
```

Comment:
Also this sounds more like a user journey, but we should also given details on the technical flow like wallet calls the mimoto and get the trusted list of issuers etc..

Reply:
Ok, sequence diagram has details about getting list of trusted issuers from mimoto. I will update in words also.